felt/Cargo.toml

```
@@ -12,6 +12,7 @@ default = ["std"]
 std = []
 alloc = []
 lambdaworks-felt = ["dep:lambdaworks-math"]
+fuzzing = ["arbitrary", "num-bigint/arbitrary"]
```

The following avoids an implicit feature:
```suggestion
fuzzing = ["arbitrary", "dep:num-bigint/arbitrary"]
```

---

I tried the change but cargo check gives me the following error:
```
Caused by:
  feature `fuzzing` includes `dep:num-bigint/arbitrary` with both `dep:` and `/`
  To fix this, remove the `dep:` prefix.
```